recipes/compass-interface-core/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+  {% set version = "0.1.4" %}
+
+package:
+  name: compass-interface-core
+  version: {{ version }}
+
+source:
+  url: https://github.com/the-scouts/compass-interface-core/releases/download/{{ version }}/compass-interface-core-{{ version }}.tar.gz
+  sha256: c102b2ea470875fa76ce1e47d805568daeb56448ff6ec8518c7f2d6201389e7b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - certifi
+    - lxml
+    - pandas
+    - python-dateutil
+    - requests
+
+test:
+  imports:
+    - compass
+
+about:
+  home: https://github.com/the-scouts/compass-interface-core
+  license: MIT
+  license_file: LICENSE
+  summary: The unofficial API to the TSA Compass membership database
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    - AA-Turner
+    - rglss
```

Comment:
@rglss Are you happy to be listed as a feedstock maintainer here?

Reply:
@chrisburr Yup, thanks for checking!

Reply:
@chrisburr also noting for avoidance of doubt I'm happy to be a maintainer here also